service/init.go

```diff
@@ -357,7 +357,9 @@ func (amf *AMF) Start() {
 		go StartGrpcServer(self.SctpGrpcPort)
 	}
 
-	go context.SetupAmfCollection()
+	if self.EnableDbStore {
+		go context.SetupAmfCollection()
+	}
```

Comment:
Thanks for change. I see in some cases EnableDbStore  check is done inside the function and sometimes its done outside. But your change is perfect. Thanks for your contribution.